### DISCUSSION TITLE
Cashout betting

### DISCUSSION THREAD

**Discussion by jbn1981** [cloudbet/docs#100] [General]
Hi, will cashout betting be available soon in the API? Thanks.

**Comment by kgravenreuth** [comment]
Hi @jbn1981 ,
Thanks for your request to add cashout bets to the public betting API. We're considering it for our roadmap, while cashout betting is not available yet on our Public Trading API. You are welcome to place cashout bets using our Web interface at [www.cloudbet.com](http://www.cloudbet.com/)

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
